internal/scheduler/database/query/query.sql

```diff
@@ -97,3 +97,16 @@ SELECT executor_id, last_updated FROM executors;
 INSERT INTO executors (executor_id, last_request, last_updated)
 VALUES(sqlc.arg(executor_id)::text, sqlc.arg(last_request)::bytea, sqlc.arg(update_time)::timestamptz)
 ON CONFLICT (executor_id) DO UPDATE SET (last_request, last_updated) = (excluded.last_request,excluded.last_updated);
+
+-- name: UpdateLeasedTime :exec
```

Comment:
I'd call these `SetX` instead of `UpdateX`.

Reply:
Actually, these queries don't make sense? We don't need a query that sets the same timestamp to multiple different runs. Instead, create a query that sets a specific timestamp for a specific run.

Reply:
> Actually, these queries don't make sense? We don't need a query that sets the same timestamp to multiple different runs. Instead, create a query that sets a specific timestamp for a specific run.

Yep, you are right. It doesn't make sense that way